src/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ set(VOXEN_SOURCES
 	src/server/world.cpp
 	src/util/exception.cpp
 	src/util/log.cpp
+	src/util/elapsed_timer.cpp
```

Comment:
This entry is out of alphabetical order - it should go before `src/util/exception.cpp`